crypto/limbs/limbs.inl

```diff
@@ -35,9 +35,24 @@
 #endif
 typedef uint8_t Carry;
 #if LIMB_BITS == 64
+#if !defined(OPENSSL_AARCH64)
```

Comment:
I think we should delete this `#if` and instead change the above:

```
#if defined(_MSC_VER) && !defined(__clang__)
```
to something like:
```
#if defined(_MSC_VER) && !defined(__clang__) && (defined(OPENSSL_X86) || defined(OPENSSL_X86_64))
```